### PR TITLE
Polish documentation a bit

### PR DESCRIPTION
**Pull request by jmid**
- Added a comment for the `Gen.sized` constructor
- Moved two FIXMEs as they were messing up the indentation of the generated ocamldoc
- Mentioned the two shorthand options for `QCheck_runner.run_tests_main`

### CODE REVIEW

File: src/QCheck.mli [hunk 1]
```diff
@@ -170,8 +170,8 @@ module Gen : sig
   val join : 'a t t -> 'a t
   (** @since NEXT_RELEASE *)
 
-  val sized : 'a sized -> 'a t
-
+  val sized : 'a sized -> 'a t (** Create a generator from a size-bounded generator *)
```
Comment (c-cube) [inline src/QCheck.mli]: I think this could be more detailed (i.e. "first it generates a random size and use it on the sized generator" or sth like this)"

**Comment by c-cube** [comment]
https://twitter.com/das_kube/status/723500462960467969 might be a good example, too, btw ;-)

**Comment by jmid** [comment]
Ok, I expanded `Gen` with ocamldoc comments for each entry.

File: src/QCheck.mli [hunk 2]
```diff
@@ -106,76 +106,120 @@ module Gen : sig
...
+      to each triple of generated elements *)
   val map_keep_input : ('a -> 'b) -> 'a t -> ('a * 'b) t
+  (** [map f g] transforms a generator [g] by applying [f] to each generated element.
+      Returns both the generated elememt from [g] and the output from [f]. *)
   val (>|=) : 'a t -> ('a -> 'b) -> 'b t
-
+  (** An infix synonym for {!map} *)
+    
   val oneof : 'a t list -> 'a t
+  (** Constructs a generator than selects among a given list of generators *)
```
Comment (c-cube) [inline src/QCheck.mli]: "that", not "than" :-)

**Comment by c-cube** [comment]
I'll fix the detail. Thanks!

**Pull request merged**